docs/Hooks.md

```diff
@@ -276,7 +276,7 @@ fastify.addHook('onRequest', (request, reply, done) => {
 fastify.addHook('preHandler', async (request, reply) => {
   await something()
   reply.send({ hello: 'world' })
-  return reply // optional in this case, but it is a good practice
+  return reply // mandatory, so the request is not executed further
```

Comment:
Is it not confusing?

The statement says that the `reply.send()` must be awaited or returned and we are avoiding both suggestions

Reply:
`reply.send() `returns the reply, so this is correct. However it would be better to have a `return reply.send()` instead.

Reply:
I modified the sentence, so it says that "reply" must be awaited or returned, not "reply.send".